tests/execution/graphql-datasource-with-empty-enum.md

```diff
@@ -0,0 +1,45 @@
+# Graphql datasource
+
+```graphql @config
+schema {
+  query: Query
+}
+
+enum EnumType {
+  INFORMATION
+  WARNING
+}
+
+type WithOptEnum {
+  type: EnumType
+}
+
+type Post {
+  severity: WithOptEnum!
+}
+
+type Query {
+  post: Post @graphQL(baseURL: "http://upstream/graphql", name: "post")
```

Comment:
We can write this test using `@expr` https://tailcall.run/docs/tailcall-dsl-graphql-custom-directives/#static

Reply:
@tusharmath updated with `@expr`